src/main/java/net/javadiscord/javabot/events/SuggestionListener.java

```diff
@@ -32,14 +34,17 @@ public void onMessageReceived(@NotNull MessageReceivedEvent event) {
 		}
 		var config = Bot.config.get(event.getGuild());
 		MessageEmbed embed = this.buildSuggestionEmbed(event.getMessage(), config.getSlashCommand());
-		this.addAttachments(event.getMessage(), event.getChannel().sendMessageEmbeds(embed)).queue(message -> {
+		this.addAttachmentsAndSend(event.getMessage(), event.getChannel().sendMessageEmbeds(embed)).thenAccept(message -> {
 					this.addReactions(message).queue();
```

Comment:
should probably put this block into a method to properly describe what's happening, also odd use of this everywhere

Reply:
What do you mean with `odd use of this`? Aside from that, what block are you talking about? The lambda body?

Reply:
yes, use methods to encapsulate that nested logic. For the this there's methods calling it prefixed with this some places, other places not. It's not a necessary call but if were going to use it that way should be standardized

Reply:
The `this` here is not context of this PR as I just left it as-is.